src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseManagementForCaseWorkerService.java

```diff
@@ -68,6 +68,7 @@ public class CaseManagementForCaseWorkerService {
     private static final String CASE_NOT_FOUND_MESSAGE = "Case Reference Number not found.";
     public static final String LISTED_DATE_ON_WEEKEND_MESSAGE = "A hearing date you have entered "
             + "falls on a weekend. You cannot list this case on a weekend. Please amend the date of Hearing ";
+    public static final int ET3_DUE_DATE_FROM_SERVING_DATE = 28;
```

Comment:
is this needed? you added it into et-common - https://github.com/hmcts/et-common/pull/248/files

Reply:
Aha, I missed that one. Fix it now. Thanks

Reply:
Fixed